pkg/actuators/machine/reconciler.go

```diff
@@ -18,24 +18,43 @@ package machine
 
 import (
 	"context"
+	"encoding/json"
 	"fmt"
+	"strings"
 	"time"
 
 	"github.com/IBM/vpc-go-sdk/vpcv1"
+	igntypes "github.com/coreos/ignition/v2/config/v3_2/types"
 	machinev1 "github.com/openshift/api/machine/v1beta1"
 	machinecontroller "github.com/openshift/machine-api-operator/pkg/controller/machine"
 	"github.com/openshift/machine-api-operator/pkg/metrics"
-	ibmcloudproviderv1 "github.com/openshift/machine-api-provider-ibmcloud/pkg/apis/ibmcloudprovider/v1"
 	apicorev1 "k8s.io/api/core/v1"
 	apimachineryerrors "k8s.io/apimachinery/pkg/api/errors"
 	klog "k8s.io/klog/v2"
 	"sigs.k8s.io/controller-runtime/pkg/client"
+
+	ibmcloudclienterrors "github.com/openshift/machine-api-provider-ibmcloud/pkg/actuators/client/errors"
+	ibmcloudproviderv1 "github.com/openshift/machine-api-provider-ibmcloud/pkg/apis/ibmcloudprovider/v1"
 )
 
 const (
 	requeueAfterSeconds      = 20
 	requeueAfterFatalSeconds = 180
 	userDataSecretKey        = "userData"
+
+	// The following values are used for machine replacement, when it appears the machine is stuck and unresponsive as part of
+	// https://issues.redhat.com/browse/OCPBUGS-1327
+	// Time in minutes a Provisioned machine has before a delete is called to force re-create
+	machineReplaceDeadline = 15
+	machineDeleteDeadline  = 10
+	// Phases and Statuses of machines
+	phaseFailed       = "Failed"
+	phaseProvisioning = "Provisioning"
+	phaseProvisioned  = "Provisioned"
+	statusRunning     = "running"
+	statusDeleting    = "deleting"
```

Comment:
is there a reason for not using the [constants from the MAO](https://github.com/openshift/machine-api-operator/blob/25dae44130fcd6d82d065340bd7ede8739c03f81/pkg/controller/machine/controller.go#L70-L92) ?

also, the running and deleting need to be capitalized

Reply:
Those constants are private, so I cannot access them I believe.

Reply:
oops, good point. let's get the capitalization fixed up

Reply:
I updated the IBM Cloud instance Statuses, but will followup on the MAO private constant changes to public in a separate PR (to isolate this mitigation to MAPI for now).